### PR TITLE
add-aditional-attributes-to-job-info

### PR DESCRIPTION
**Pull request by oanaBiancaSchiopu**

### CODE REVIEW

**Comment by fviale** [comment]
@oanaBiancaSchiopu we should wait for the merge of the scheduling-api pull request.

**Comment by oanaBiancaSchiopu** [comment]
retest this please

**Review by mboussaa — APPROVED** [review]

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/GraphQLQueries.java [hunk 1]
```diff
@@ -477,6 +485,104 @@ else if (!value.equals(projectName))
...
+                                break;
+                            default:
+                                break;
+                        }
+                        break;
+                    }
+                    case PARENT_ID: {
+                        // Consider only parseable numbers.
+                        try {
+                            dateInMs = Integer.parseInt(value);
```
Comment (fviale) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/GraphQLQueries.java:518]: variable should not be named dateInMs

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/GraphQLQueries.java [hunk 2]
```diff
@@ -477,6 +485,104 @@ else if (!value.equals(projectName))
                         }
                         break;
                     }
+                    case CUMULATED_CORE_TIME: {
+                        // Consider only parseable numbers.
+                        try {
+                            dateInMs = Long.parseLong(value);
```
Comment (fviale) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/GraphQLQueries.java:491]: variable should not be named dateInMs

**Review by fviale — APPROVED** [review]
LGTM, one minor comment

**Pull request merged**